pkg/ivd/ivd_protected_entity_type_manager.go

```diff
@@ -346,8 +346,8 @@ func (this *IVDProtectedEntityTypeManager) ReloadConfig(ctx context.Context, par
 			// Disconnecting older vc instance.
 			err = this.vcenter.Disconnect(ctx)
 			if err != nil {
-				this.logger.Errorf("Failed to disconnect older vcenter instance.")
-				return err
+				this.logger.Warnf("Failed to disconnect older vcenter instance," +
+					" ignoring and proceeding, err: %+v", err)
```

Comment:
What's the side effect for ignoring the disconnect error?  Does it prevent the plugin from reconnecting?

Reply:
Lintong looked at logs that proved this is the right fix.